source/code/scxsystemlib/disk/diskdepend.cpp

```diff
@@ -1438,6 +1438,7 @@ namespace SCXSystemLib
 
         if (SolarisPartitionNamePatternRegExPtr->IsMatch(path.GetFilename()))
         {    // remove the "p#", "s#", "p##" or "s##" from the end of the name
+
```

Comment:
Any reason for this? In general, we don't do blank lines after a { ... we just go on the next line (indented).